docs/lineage/prefect.md

```diff
@@ -0,0 +1,57 @@
+# Prefect Integration
+
+DataHub supports integration of
+
+- Prefect flow and task metadata
+- Flow run and Task run information as well as
+- Lineage information when present
+
+## What is Prefect Datahub Block?
+
+Blocks are primitive within Prefect that enable the storage of configuration and provide an interface for interacting with external systems. We integrated [prefect-datahub](https://prefecthq.github.io/prefect-datahub/) block which use [Datahub Rest](../../metadata-ingestion/sink_docs/datahub.md#datahub-rest) emitter to emit metadata events while running prefect flow.
+
+## Prerequisites to use Prefect Datahub Block
+
+1. You need to use either Prefect Cloud (recommended) or the self hosted Prefect server.
+2. Refer [Cloud Quickstart](https://docs.prefect.io/2.10.13/cloud/cloud-quickstart/) to setup Prefect Cloud.
+3. Refer [Host Prefect server](https://docs.prefect.io/2.10.13/host/) to setup self hosted Prefect server.
+4. Make sure the Prefect api url is set correctly. You can check it by running below command:
+```shell
+prefect profile inspect
+```
+5. If you are using Prefect Cloud, the api url should be set as `https://api.prefect.cloud/api/accounts/<account_id>/workspaces/<workspace_id>`.
```

Comment:
If you are using Prefect Cloud, the API URL should be set as `https://api.prefect.cloud/api/accounts/<account_id>/workspaces/<workspace_id>`.